internal/checkers/negative_postive.go

```diff
@@ -0,0 +1,200 @@
+package checkers
+
+import (
+	"go/ast"
+	"go/token"
+
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/Antonboom/testifylint/internal/analysisutil"
+)
+
+// NegativePositive detects situations like
+//
+//	assert.Greater(t, a, 0)
+//	assert.Less(t, 0, a)
+//	assert.True(t, a > 0)
+//	assert.True(t, 0 < a)
+//	assert.False(t, a <= 0)
+//	assert.False(t, 0 >= a)
+//
+//	assert.Less(t, a, 0)
+//	assert.Greater(t, 0, a)
+//	assert.True(t, a < 0)
+//	assert.True(t, 0 > a)
+//	assert.False(t, a >= 0)
+//	assert.False(t, 0 <= a)
+//
+// and requires
+//
+//	assert.Positive(t, value)
+//	assert.Negative(t, value)
+type NegativePositive struct{}
+
+// NewNegativePositive constructs NegativePositive checker.
+func NewNegativePositive() NegativePositive { return NegativePositive{} }
+func (NegativePositive) Name() string       { return "negative-positive" }
+
+func (checker NegativePositive) Check(pass *analysis.Pass, call *CallMeta) *analysis.Diagnostic {
```

Comment:
implementation is consistent with `Empty` checker

Reply:
Can you point out what you meant about this ? I'm unsure what you meant

Reply:
I meant a code structure: help methods like checker.checkNegative/Positive, switches inside, etc.

Reply:
Thanks, I now get it. Yes, it's clearer and consistent.